src/views/Nft/market/Profile/UnconnectedProfile.tsx

```diff
@@ -31,7 +31,7 @@ const UnconnectedProfile = () => {
   const { profile: profileHookState, isFetching: isProfileFetching } = useProfileForAddress(accountAddress)
   const { profile } = profileHookState || {}
   const { achievements } = useAchievementsForAddress(accountAddress)
-  const { nfts, isLoading } = useGetNftsForAddress(accountAddress, profile, isProfileFetching)
+  const { nfts, isLoading } = useNftsForAddress(accountAddress, profile, isProfileFetching)
```

Comment:
Change the name to make it look like the other fetch functions (achievements, profiles)